src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -250,6 +256,9 @@ def __call__(
             )
             uncond_embeddings = self.text_encoder(uncond_input.input_ids.to(self.device))[0]
 
+            # duplicate unconditional embeddings for each generation per prompt
+            uncond_embeddings = uncond_embeddings.repeat_interleave(batch_size * num_images_per_prompt, dim=0)
```

Comment:
Last time I checked I had issues with `repeat_interleave` and ONNX. We probably just need to require a specific opset version though.
Can you please double-check @anton-l?

Reply:
Thanks, will check that. But this PR doesn't modify onnx pipeline, so it should be fine.